src/math/bigintops.c

```diff
@@ -370,8 +376,12 @@ MVMObject * MVM_bigint_##opname(MVMThreadContext *tc, MVMObject *result_type, MV
     ia = force_bigint(tc, ba, 0); \
     ib = force_bigint(tc, bb, 1); \
     ic = MVM_malloc(sizeof(mp_int)); \
-    mp_init(ic); \
-    mp_##opname(ia, ib, ic); \
+    if ((err = mp_init(ic)) != MP_OKAY) { \
+        MVM_exception_throw_adhoc(tc, "Error initializing a big integer: %s", mp_error_to_string(err)); \
+    } \
+    if ((err = mp_##opname(ia, ib, ic)) != MP_OKAY) { \
+        MVM_exception_throw_adhoc(tc, "Error performing ##opname with big integers: %s", mp_error_to_string(err)); \
```

Comment:
`##opname` won't be replaced by the preprocessor in a string, you should either do `"Error performing "##opname" with big integers: %s"` or `"Error performing %s with big integers: %s", ##opname ...` where I'd prefer the first